### ISSUE TITLE
MicroProfile 4.1 Compatible Certification Request

### ISSUE THREAD

**Issue by Emily-Jiang** [microprofile/microprofile-wg#95]
- [x] Organization Name ("Organization") and, if applicable, URL:<br/>
IBM
- [x] Product Name, Version and download URL (if applicable):<br/>
[Open Liberty 21.0.0.8-beta](https://repo1.maven.org/maven2/io/openliberty/beta/openliberty-runtime/21.0.0.8-beta/openliberty-runtime-21.0.0.8-beta.zip) [Source jar](https://github.com/OpenLiberty/open-liberty/releases/tag/gm-21.0.0.8-beta)
- [x] Specification Name, Version and download URL:<br/>
 MicroProfile 4.1
- [x] Public URL of TCK Results Summary:<br/>
[MicroProfile 4.1 TCK results](https://openliberty.io/certifications/microprofile/4.1/TCKResults.html)
- [x] Any Additional Specification Certification Requirements:<br/>
No
- [x] Java runtime used to run the implementation:<br/>
Java 8, 11
- [x] Summary of the information for the certification environment, operating system, cloud, ...:<br/>
See the TCK results for details
- [x] By checking this box I acknowledge that the Organization I represent accepts the terms of the [EFTL](https://www.eclipse.org/legal/tck.php).
- [x] By checking this box I attest that all TCK requirements have been met, including any compatibility rules.

**Comment by edbratt** [comment]
The TCK summary page references Jakarta EE TCK results page, that describe tests against a different version of OpenLiberty. If this version intends to claim compatibility with Jakarta EE TCKs, it should include proper evidence for this version.

I am not clear what requirements were settled on, for compatibility claims with MicroProfile 4.1 and the required Jakarta EE component specifications but, for me, it would be better to just omit these since they are not really relevant. If there are results for 21.0.0.8-Beta, it would be excellent to include them.

In short: I would either drop this reference to 20.0.0.3, or provide a reference for 21.0.0.8-beta.

**Comment by Emily-Jiang** [comment]
@edbratt thanks for reviewing this! The settlement on the compatibility was in the MP 4.1 spec itself: only require passing the 8 MP spec tcks. I will drop the reference to 20.0.0.3 for Jakarta EE tck results to avoid any confusion. I'll get this done on Monday as I will be off tomorrow.

**Comment by Emily-Jiang** [comment]
Before I log off for my PTO, I managed to do the changes to remove the Optional Jakarta EE section and hopefully the change will be visible tomorrow.
